deal.II-toolchain/packages/gsl.package

```diff
@@ -0,0 +1,59 @@
+################################################################################
+## GSL                                                                        ##
+################################################################################
+
+VERSION=release-2-6
+NAME=gsl.git
+EXTRACTSTO=gsl-${VERSION}
+SOURCE=https://git.savannah.gnu.org/git/
+PACKING=git
+BUILDCHAIN=custom
+
+BUILDDIR=${BUILD_PATH}/gsl-${VERSION}
+INSTALL_PATH=${ORIG_INSTALL_PATH}/gsl-${VERSION}
+
+
+
+CONFOPTS=""
+
+
+
+package_specific_build () {
```

Comment:
As far as I can see, we can use
`package_specific_setup () { ./autogen.sh }` and set
`CONFOPTS="--enable-maintainer-mode"`

I check that and write you.